### PR TITLE
104 introduce best practices document

### PR DESCRIPTION
**Pull request by ssteinbuss**
As discussed during the last meetings the PR contains an initial structure to create a best practices document. 

* The README.md of the specification contains a teaser for the Best Practices
* A generic introduction in the README.md of the best practices
* The following sections:
   * Related documents
   * Examples
   * How to extend the dataspace protocol
   * How to create a new binding
   * Known implementations
   * Security considerations
* Lined the best practices to the GitBook rendering

### CODE REVIEW

File: best.practices/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The key words *MAY*, *MUST*, and *MUST NOT* in this document are to be interpreted as described in [RFC2119](https://www.rfc-editor.org/rfc/rfc2119) and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174) when, and only when, they appear in all capitals, as shown here.
+
+## Introduction
+
+The [***Dataspace Protocol***](../README.md) is a set of specifications designed to facilitate interoperable data sharing between entities governed by usage control and based on Web technologies. These specifications
+define the schemas and protocols required for entities to publish data, negotiate usage agreements, and access data as part of a federation of technical systems termed a
+***dataspace***.
+
+The Dataspace Protocol is under development and the working group is active on this draft, reviewed and improved the content multiple times. During the process several aspects were discussed, which are not considered relevant for the specification itself, but important to be documented as support for the users of this specification as best practices. The in this document are non-normative.
```
Comment (juliapampus) [inline best.practices/README.md:25]:
```suggestion
The Dataspace Protocol is under development and the working group is active on this draft, reviewed and improved the content multiple times. During the process several aspects were discussed, which are not considered part of the normative specification, but important to be documented as support for the users of this specification as best practices. The in this document are non-normative.
```
Well, they are relevant, but not part. Maybe rephrase?

Comment (juliapampus) [inline best.practices/README.md:25]: The last sentence is missing a word

File: README.md [hunk 2]
```diff
@@ -57,3 +57,16 @@ A connector will implement additional internal functionalities, like monitoring
 The same applies for the data, which is transferred between the systems. While this document does not define the transport protocol, the structure, syntax and semantics of the data, a specification for those aspects is required and subject to the agreements of the participants or the data space.
 
 ![Overview on protocol and context](./resources/figures/ProtocolOverview.png)
+
+## Best Practices
+
+The Dataspace Protocol is under development and the working group is active on this draft, reviewed and improved the content multiple times. During the process several aspects were discussed, which are not considered relevant for the specification itself, but important to be documented as support for the users of this specification as best practices. The [Best Practices](./best.practices/README.md) are non-normative.
```
Comment (juliapampus) [inline README.md:63]:
```suggestion
The Dataspace Protocol is under development and the working group is active on this draft, reviewed and improved the content multiple times. During the process several aspects were discussed, which are not considered part of the normative specification, but important to be documented as support for the users of this specification as best practices. The [Best Practices](./best.practices/README.md) are non-normative.
```

**Review by juliapampus — APPROVED** [review]
Rest looks good to me

**Comment by sebbader-sap** [comment]
@ssteinbuss please check the comments and either integrate them or merge directly.
